rules/endpoints.go

```diff
@@ -270,6 +280,7 @@ func (r *DefaultRuleRenderer) endpointIptablesChain(
 	adminUp bool,
 	allowAction Action,
 	dropEncap bool,
+	defaultDropWhenNoPolicy bool,
```

Comment:
This method is getting unwieldy with the number of flag options.  Using an args struct might be cleaner at this point

Reply:
Is something like this what you had in mind? Maybe there's a better way of doing this
```
func (r *DefaultRuleRenderer) HostEndpointToRawChains(
	ifaceName string,
	ingressPolicyNames []string,
	egressPolicyNames []string,
) []*Chain {
	log.WithField("ifaceName", ifaceName).Debug("Rendering raw (untracked) host endpoint chain.")

	toOptions := &endpointIptablesChainOptions{
		ifaceName,
		PolicyOutboundPfx,
		ProfileOutboundPfx,
		HostToEndpointPfx,
		ChainFailsafeOut,
		chainTypeUntracked,
		true, // Host endpoints are always admin up.
		AcceptAction{},
		dontDropEncap,
		false,
	}
	// Chain for traffic _to_ the endpoint.
	chainTrafficToEndpoint := r.endpointIptablesChain(
		egressPolicyNames,
		nil, // We don't render profiles into the raw table.
		toOptions,
	)

	fromOptions := &endpointIptablesChainOptions{
		ifaceName,
		PolicyInboundPfx,
		ProfileInboundPfx,
		HostFromEndpointPfx,
		ChainFailsafeIn,
		chainTypeUntracked,
		true, // Host endpoints are always admin up.
		AcceptAction{},
		dontDropEncap,
		false,
	}
	// Chain for traffic _from_ the endpoint.
	chainTrafficFromEndpoint := r.endpointIptablesChain(
		ingressPolicyNames,
		nil, // We don't render profiles into the raw table.
		fromOptions,
	)

	return []*Chain{
		chainTrafficToEndpoint,
		chainTrafficFromEndpoint,
	}
}
```

Reply:
Yes, except the point was to use the names of the struct fields to make it clear what all the args are (and to allow for omitting args that have a sensible default)

Reply:
d064081b976f9713133e160f4477926b00d0e461

Reply:
I removed this refactoring since we ended up not using the extra function arg.